src/main/java/org/wiremock/integrations/testcontainers/WireMockContainer.java

```diff
@@ -184,15 +182,18 @@ public WireMockContainer withExtension(String id, String className) {
         return withExtension(id, Collections.singleton(className), Collections.emptyList());
     }
 
-    public String getEndpoint() {
-        return String.format("http://%s:%d", getHost(), getMappedPort(PORT));
+    public String getBaseUrl() {
+        return String.format("http://%s:%d", getHost(), getPort());
     }
 
-    public URI getRequestURI(String relativePath) throws URISyntaxException {
-        return new URI(getEndpoint() + "/" + relativePath);
+    public String getUrl(String path) {
+        if (!path.startsWith("/")) {
+            path = "/" + path;
+        }
+        return String.format("%s%s", getBaseUrl(), path);
     }
 
-    public Integer getServerPort() {
+    public int getPort() {
```

Comment:
It will lead to NPE if `null` is actually returned https://www.tabnine.com/code/java/methods/org.testcontainers.containers.GenericContainer/getMappedPort

Reply:
Agree, it is better to use type`Integer`